### DISCUSSION TITLE
Welcome to json-csv Discussions!

### DISCUSSION THREAD

**Discussion by ssotka** [ssotka/json-csv#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

 I'm the repo owner and I approve this boilerplate.
-Scott

**Comment by roncbailey** [comment]
Hey Scott,  you got a call out on https://rakudoweekly.blog/2022/07/18/2022-29-hot/. Rakudo weekly.

**Reply by ssotka** [reply]
I saw that! Awesome
